recipes/pslibrary/meta.yaml

```diff
@@ -0,0 +1,1240 @@
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Cs.rel-pz-spnl-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Be.rel-pz-s-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Gd.rel-pz-spdfn-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Ar.rel-pz-nl-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Pu.rel-pz-spfn-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Cu.rel-pz-spn-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/P.rel-pz-nl-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Ti.rel-pz-spn-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Ba.rel-pz-spn-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Au.rel-pz-spfn-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Hf.rel-pz-spdfn-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Eu.rel-pz-spdfn-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Cd.rel-pz-n-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Co.rel-pz-n-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Sn.rel-pz-dn-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Cr.rel-pz-spn-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Ar.rel-pz-n-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/O.rel-pz-nl-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Rb.rel-pz-spn-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Pu.rel-pz-spfn-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Mg.rel-pz-spnl-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Hg.rel-pz-n-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Sb.rel-pz-n-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Tc.rel-pz-spn-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/P.rel-pz-n-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Zr.rel-pz-spn-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Fe.rel-pz-spn-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Cl.rel-pz-nl-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Cr.rel-pz-spn-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Ce.rel-pz-spdfn-rrkjus_psl.1.0.0.UPF
```

Comment:
Can we be smarter about this and avoid a recipe with thousand of line? I don't like fancy jinjas and look but maybe this a case where we should use them.

Reply:
I could move the tests to a separate file, but I am not exactly sure how to solve it inside the yaml file with more fancy jinjas.

Reply:
To be honest I never wrote a jinja to do that, not even sure how to do it myself.
Moving to another file is a good compromise until we have a better solution.